tests/unit/core/reducers/test_addons.js

```diff
@@ -310,78 +323,14 @@ describe(__filename, () => {
 
     it('returns an add-on by id', () => {
       const { store } = dispatchClientMetadata();
-      store.dispatch(loadAddon({ addon: fakeAddon }));
+      store.dispatch(loadAddon({ addon: fakeAddon, slug: fakeAddon.slug }));
 
       expect(getAddonByID(store.getState().addons, fakeAddon.id)).toEqual(
         createInternalAddon(fakeAddon),
       );
     });
   });
 
-  describe('getAddonBySlug', () => {
```

Comment:
Do we need any tests for the new `getAddonByIdInURL` function?

Reply:
ha maybe yes